dbt/models/iasworld/schema/iasworld.dweldat.yml

```diff
@@ -42,6 +42,20 @@ sources:
             description: '{{ doc("column_areasum") }}'
           - name: attic
             description: '{{ doc("shared_column_char_attic_type") }}'
+            tests:
+              - not_null:
+                  name: iasworld_dweldat_attic_not_null
+                  # Excludes non-regression classes, as defined in Assessor's
+                  # online PDF of all class definitions
+                  config: &unique-conditions
+                    where: | 
+                      cur = 'Y'
+                      AND deactivat IS NULL
+                      AND class NOT IN ('201', '213', '218', '219', '220', '221', '224', '225', '236', '240', '241', '290', '294', '297')
```

Comment:
`DWELDAT` has a bunch of non-regression classes that don't actually have any characteristic data, hence the exclusion here.